### DISCUSSION TITLE
How to set proxy when download sdks

### DISCUSSION THREAD

**Discussion by vivostar** [sdkman/sdkman-cli#1191] [Q&A] [answered]
In china, we have to use vpn or proxy to visite maven repositoy, github and other website not in China. sdkman is a useful tool to manager jdks and sdks. but I want to set proxy when downloading sdks(the downloading is very slow for China's gfw). Could you give me some suggestions?

**Comment by marc0der** [comment, marked as answer]
Hi @vivostar 👋

SDKMAN uses `curl` to make all http calls, so you can configure it in the usual way to use your proxy.

This article has some good tips to configure `curl`:
https://www.baeldung.com/linux/curl-permanent-proxy

Hope it helps!

**Reply by vivostar** [reply]
thanks, @marc0der, thank you very much.
